frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -2889,6 +2889,54 @@ static bool isKeywordTypeCtor(Context* context,
   return false;
 }
 
+static void buildReaderWriterTypeCtor(Context* context,
+                                      const CallInfo& ci,
+                                      const TypedFnSignature* initial,
+                                      CandidatesVec& initialCandidates) {
+  std::vector<UntypedFnSignature::FormalDetail> formals;
+  // Move 'kind' to the end and allow the first two args to just be
+  // 'locking' and  '(de)serializerType'
```

Comment:
Is this sufficient because `kind` has a default value?

Reply:
Yes, the field now has a default value when it did not before.